client/src/components/Homepage/indexCSS.css

```diff
@@ -1,5 +1,5 @@
 body {
```

Comment:
Try changing this global CSS to .some_class and then use that class in the file. Also it isn't conflicting right?